From adfbf9e6cad93281cffceab078e7f6f2a8e094f9 Mon Sep 17 00:00:00 2001
From: Alexey Brodkin <abrodkin@synopsys.com>
Date: Thu, 13 Aug 2015 01:56:02 +0300
Subject: [PATCH 1/2] openwrt: arc - remove dependency on DEVTMPFS

OpenWRT builds initramfs so that it doesn't require DEVTMPFS so dropping
this dependency. That helps to escape 2 separate kernel rebuilds with
and without initramfs.

2 builds happen because OpenWRT first builds kernel and later modules.
When building entire kernel with simple "make" INITRAMFS sets to a real
value and so was triggering DEVTMPFS selection. Then when building only
modules with "make modules" command INITRAMFS is zeroed and so kernel
config was changing that lead to full kernel rebuild.

Signed-off-by: Alexey Brodkin <abrodkin@synopsys.com>
---
 arch/arc/Kconfig | 2 --
 1 file changed, 2 deletions(-)

--- a/arch/arc/Kconfig
+++ b/arch/arc/Kconfig
@@ -12,8 +12,6 @@ config ARC
 	select BUILDTIME_EXTABLE_SORT
 	select COMMON_CLK
 	select CLONE_BACKWARDS
-	# ARC Busybox based initramfs absolutely relies on DEVTMPFS for /dev
-	select DEVTMPFS if !INITRAMFS_SOURCE=""
 	select GENERIC_ATOMIC64
 	select GENERIC_CLOCKEVENTS
 	select GENERIC_FIND_FIRST_BIT
